index.html

```diff
@@ -15,102 +15,117 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400&display=swap" rel="stylesheet">
 </head>
 <body>
-    <header>
-        <nav class="container">
-            <ul>
-                <li><h2 class="baritem">Welcome</h2></li>          
-                <li><a href="/" rel="noopener"><img class="hamburger" src="./Union.svg" alt="menu" ></li></a>
+    <!-- esto es la zona alta del desktop version -->
+    <!-- esta parte es la parte normal de la version movil -->
+        <header>
+            <nav class="container">
+                <ul>
+                    <li><h2 class="baritem">Welcome</h2></li>          
+                    <li><a href="/" rel="noopener"><img class="hamburger" src="./Union.svg" alt="menu" ></li></a>
+                </ul>
+            </nav>
+        </header>
+        <div class="containerDesk">
+            <ul class="unlistbar">
+                <li class="text">portfolio</li>
+                <li class="text">about</li>
+                <li class="text">contact</li>
             </ul>
-        </nav>
-    </header>
-    <section class="tittleContainer">
-        <h1 class="welcome">Hey there. I'm Pierre barba<br><span class="description"> I'm a software developer</span></h1>
-        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-       </p>
-        <ul class="mediaContainer">
-            <li><a href="https://twitter.com/magpe02" target="_blank" rel="noopener" ><img class="mediaIcon" src="./twitter.svg" alt="Twitter"></a></li>
-            <li><a href="https://www.linkedin.com/in/jean-pierre-barba-arredondo-114b851b4/" target="_blank" rel="noopener" ><img class="mediaIcon" src="./Linkedin icon.svg" alt="linkedin"></a></li>
-            <li><a href="https://medium.com/" target="_blank" rel="noopener" ><img class="mediaIcon" src="./medium (1).svg" alt="github"></a></li>
-            <li><a href="https://github.com/PierreBarba" target="_blank" rel="noopener" ><img class="mediaIcon" src="./github.svg" alt="Twitter"></a></li>
-            <li><a href="https://angel.co/" target="_blank" rel="noopener"><img class="mediaIcon" src="./angellist.svg" alt="github"></a></li>
-        </ul>
-    </section>
-    <a class="arrowDown" href="#MyRecentWork"><img src="./Disabled.svg" alt="down arrow"></a>
+        </div>
+        <section class="tittleContainer">
+            <!-- <div class="imgBackground">
+                <img class="imgOneMovil" src="./header-illsutration-mobile@2x.svg" alt="imgBackground">
+                <img class="imgTwoDesk" src="./imgdesktopversion/Header-llustration-desktop.svg" alt="otherImg">
```

Comment:
- [ ] Please do not forget to remove this commented code, to help you get a cleaner code base.